Source/player.cpp

```diff
@@ -1473,8 +1474,7 @@ void ValidatePlayer()
 	assert(MyPlayer != nullptr);
 	Player &myPlayer = *MyPlayer;
 
-	if (myPlayer._pLevel > MaxCharacterLevel)
-		myPlayer._pLevel = MaxCharacterLevel;
+	myPlayer.setCharacterLevel(myPlayer.getCharacterLevel());

```

Comment:
From the first look we read the player level and set it, that doesn't make sense (for the first look).
The function ensures that player level is valid, but for clarity I would add a comment or keep the original if else this is confusing (at least for me).

Reply:
good point, I wasn't particularly happy about this function existing in the first place (seems overkill to check stats have been set correctly on every frame) so I didn't put much thought into commenting what it was doing.